### ISSUE TITLE
Undefined class constant 'ROOT_PACKAGE_NAME' in src/DependencyInjection/Configuration.php:115

### ISSUE THREAD

**Issue by Tanuel** [getsentry/sentry-symfony#331]
this package has no direct dependency to `ocramius/package-versions`, but tries to access a class constant not present in the minimum version set by `jean85/pretty-package-versions`

simply running `composer update` does not help.

An easy fix for this issue is to actually require `ocramius/package-versions` in `composer.json` instead of depending on another dependency to do that.

[version 1.3.0](https://github.com/Ocramius/PackageVersions/releases/tag/1.3.0) is required for ROOT_PACKAGE_NAME but jean85/pretty-package-versions brings 1.2

```bash
composer why ocramius/package-versions
jean85/pretty-package-versions  1.2  requires  ocramius/package-versions (^1.2.0)  
```

**Comment by Jean85** [comment]
That's right and it WAS like that. I did a mistake with #325 in https://github.com/getsentry/sentry-symfony/commit/d48d50dfdcb93df1eb00719418a43f5976743f71#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780L24

That should be reverted. We could add `composer-require-checker` in CI to avoid those issues in the future. In the meantime, `composer update` should fix your issue. If it doesn't, try running `composer why-not ocramius/package-versions:^1.3` to debug.

**Comment by Tanuel** [comment]
> In the meantime, `composer update` should fix your issue. If it doesn't, try running `composer why-not ocramius/package-versions:^1.3` to debug.

For some reason, composer update will not update to the latest, but the minimum required version.
```bash
composer why-not ocramius/package-versions:^1.3
There is no installed package depending on "ocramius/package-versions" in versions not matching ^1.3
```

what will solve this issue is running `composer require ocramius/package-versions"

**Comment by Jean85** [comment]
You surely have some issues on your end, because that package is a strong requirement for `jean85/pretty-package-versions`, it's not possibile to NOT have it in some version, at least.

**Comment by Tanuel** [comment]
Yes I had it in the lowest possible version which was 1.2. there might be some weirdness with our satis repository tho which I can try to figure out.
What I just find weird is that composer require did not add the package to the composer.json requirements...

**Comment by Jean85** [comment]
Fix released as 3.5.0: https://github.com/getsentry/sentry-symfony/releases/tag/3.5.0
